src/OctoshiftCLI.Tests/ado2gh/Commands/MigrateRepo/MigrateRepoCommandTests.cs

```diff
@@ -33,43 +33,19 @@ public void Should_Have_Options()
         {
             _command.Should().NotBeNull();
             _command.Name.Should().Be("migrate-repo");
-            _command.Options.Count.Should().Be(12);
+            _command.Options.Count.Should().Be(11);
 
             TestHelpers.VerifyCommandOption(_command.Options, "ado-org", true);
             TestHelpers.VerifyCommandOption(_command.Options, "ado-team-project", true);
             TestHelpers.VerifyCommandOption(_command.Options, "ado-repo", true);
             TestHelpers.VerifyCommandOption(_command.Options, "github-org", true);
             TestHelpers.VerifyCommandOption(_command.Options, "github-repo", true);
             TestHelpers.VerifyCommandOption(_command.Options, "ado-server-url", false, true);
-            TestHelpers.VerifyCommandOption(_command.Options, "wait", false, true);
             TestHelpers.VerifyCommandOption(_command.Options, "queue-only", false);
             TestHelpers.VerifyCommandOption(_command.Options, "target-repo-visibility", false);
             TestHelpers.VerifyCommandOption(_command.Options, "ado-pat", false);
             TestHelpers.VerifyCommandOption(_command.Options, "github-pat", false);
             TestHelpers.VerifyCommandOption(_command.Options, "verbose", false);
         }
-
-        [Fact]
-        public void It_Uses_Github_Pat_When_Provided()
-        {
-            var adoPat = "abc123";
-            var githubPat = "def456";
-
-            var args = new MigrateRepoCommandArgs
-            {
-                AdoOrg = "foo-org",
-                AdoTeamProject = "blah-tp",
-                AdoRepo = "some-repo",
-                GithubOrg = "gh-org",
-                GithubRepo = "gh-repo",
-                Wait = true,
-                AdoPat = adoPat,
-                GithubPat = githubPat,
-            };
-
-            _command.BuildHandler(args, _serviceProvider);
-
-            _mockGithubApiFactory.Verify(m => m.Create(null, githubPat));
-        }
```

Comment:
Did you mean to delete this test? It seems like this test should still be relevant.